src/main/java/com/uid2/shared/vertx/VertxUtils.java

```diff
@@ -78,6 +79,21 @@ public static ConfigRetriever createConfigRetriever(Vertx vertx) {
         return ConfigRetriever.create(vertx, retrieverOptions);
     }
 
+    public static AbstractMap.SimpleEntry<String, String> parseClientAppVersion(String appVersions) {
+        final int eqpos = appVersions.indexOf('=');
```

Comment:
For reference, what's a sample `appVersions` string? A unit test for this might be helpful

Reply:
Added tests